CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## Unreleased
 - Improve sats amount readability adding thousands separator
 - Update minimum supported buntu version to 20.04+
+- Hide 0 amount ERC20 txs to partially mitigate Address Poisoning Attack
```

Comment:
txs=>transactions. Imo no point in abbreviating where not necessary, as there are always people that don't know some acronyms or short names.